guide/daily-spending-wallet/security.md

```diff
@@ -278,7 +232,7 @@ However, this prevents mobile lightning wallets from being reliable monitors of
 
 One technique to reduce this risk is to notify the user when the application has been offline for an extended period. This way, they can open the app and thereby enable it to check their channels. Forced channel closes have a built-in, agreed-upon delay. Send notifications earlier than this delay so that users have time to respond appropriately.
 
-Another technique is to use [watchtowers](https://wiki.ion.radar.tech/tech/research/watchtowers). These are third parties that continuously monitor wallets and punish bad actors who attempt to cheat the wallet they are monitoring. Watchtowers should be provided by a different party than the wallet and payment channels to reduce the risk of collusion.
+Another technique is to use [watchtowers](https://wiki.ion.radar.tech/tech/research/watchtowers). These are third parties that continuously monitor wallets and punish bad actors who attempt to cheat the wallet they are monitoring. Watchtowers should be provided by a different party than the wallet and payment channels, to reduce the risk of collusion.
 
 {% include image-gallery.html pages = page.images_watchtower %}
 
```

Comment:
Good review overall. Some suggestions of stuff to add:

- Make it so users can't screenshot recovery phrases when manually backing up (or other sensitive info)
- Scrambled PIN
- Option for users to securely erase wallet (say if they are moving from one phone to another)

Reply:
What do you mean by scrambled PIN?

Reply:
Scrambles / randomizes the layout of the PIN numbers. If a camera or someone's sees your pattern of entry they can quite easily know the PIN entered. 

Example in Samourai below. Zeus also offers this

![image](https://user-images.githubusercontent.com/55287964/179701411-e7fe6ac8-615a-4a73-83a3-e0ee10cf8a0c.png)

Reply:
Not being able to screenshot the recovery phrase is a good addition to the manual backup page.

I just added scrambled PINs and erasing the wallet to the page.